packager/media/codecs/h265_byte_to_unit_stream_converter.cc

```diff
@@ -72,8 +72,10 @@ bool H265ByteToUnitStreamConverter::GetDecoderConfigurationRecord(
   buffer.AppendInt(static_cast<uint8_t>(kUnitStreamNaluLengthSize - 1));
   buffer.AppendInt(static_cast<uint8_t>(3) /* numOfArrays */);
 
+  // More parameter set NALUs may follow when strip_parameter_set_nalus is disabled.
+  const uint8_t kArrayCompleteness = strip_parameter_set_nalus() ? 0x80 : 0;
```

Comment:
Style nit: Since this is no longer a compile-time constant, it should be renamed to something like array_completeness.